deps/obs-scripting/obs-scripting-python-import.c

```diff
@@ -116,6 +118,7 @@ bool import_python(const char *python_path, python_version_t *python_version)
 
 	IMPORT_FUNC(PyType_Ready);
 	IMPORT_FUNC(PyType_Modified);
+	IMPORT_FUNC(PyType_IsSubtype);
```

Comment:
Is this necessary for this bugfix?

Reply:
It could lead to crashes at runtime if we don't import the actual function, as the aliased function pointer will not point to a valid memory address.

Reply:
Fair enough.